upstream/solr6/local/92-init-solr.sh

```diff
@@ -9,7 +9,7 @@ echo "Solr init start"
 
 SOLR_DATA_ROOT="$SOLR_INSTALL_HOME/data"
 DIR_ROOT=${DIR_ROOT:-'/opt/alfresco-search-services/data'}
-SOLR_BACKUP_DIR=${SOLR_BACKUP_DIR:-'/opt/alfresco-search-services/data/solr6Backup'}
+SOLR_BACKUP_DIR=${SOLR_BACKUP_DIR:-'/opt/alfresco-search-services/backup'}
```

Comment:
don't change the default, this is a breaking change

Reply:
what about this then https://github.com/xenit-eu/helm-alfresco/pull/57/files#r1154154470 ?